src/main/java/com/libertymutualgroup/herman/aws/ecs/broker/iam/IAMBroker.java

```diff
@@ -69,11 +73,14 @@ public Role brokerAppRole(AmazonIdentityManagement client, IamAppDefinition defi
         if (role == null) {
             buildLogger.addLogEntry("... Creating new role: " + definition.getAppName());
 
+            List<com.amazonaws.services.identitymanagement.model.Tag> iamTags = TagUtil.hermanToIamTags(tags);
+
             CreateRoleRequest createRoleRequest = new CreateRoleRequest()
                 .withPath("/aws-ecs/")
                 .withRoleName(definition.getAppName())
                 .withAssumeRolePolicyDocument(assumePolicy);
             client.createRole(createRoleRequest);
+            client.tagRole(new TagRoleRequest().withTags(iamTags));
```

Comment:
Could you actually put this line down lower outside the condition? Or at least include it in the else that goes with this block as well, so that tags get updated when a role already exists and aren't just set on create.